src/components/SectionDividier/Divider.js

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import PropTypes from "prop-types";
+import Spacing, { spacingShape } from "../Spacing";
+import BOTTOM_SPACING, { GENERAL_CONTENT_SPACER } from "./constant";
+
+const Divider = ({ top, bottom, type, children }) => {
```

Comment:
Let's add some comments on why do we still need the <Spasing top.. . bottom ... > case. Mention that it's just for the backward compatibility with the current implementation and we age going to remove this part after cleaning the appropriate feature flag.

Reply:
Done.